### PR TITLE
missing man pages for ceph-create-keys, ceph-disk-*

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/11862

### CODE REVIEW

**Comment by smithfarm** [comment]
@tchaikov can you take a look?

**Comment by tchaikov** [comment]
once it is also added into the rpm spec file. and confirmed by the gitbuilders. `Reviewed-by`.

@smithfarm i'd suggest you to push it to a wip branch to get it tested with the tireless builders.

**Comment by smithfarm** [comment]
@tchaikov: I noticed that http://tracker.ceph.com/issues/10725 is asking for 4 manpages, but it looks like you resolved it after adding only this one . . . (?) Should we open issues for the remaining 3?

**Comment by tchaikov** [comment]
i don't think so. see http://tracker.ceph.com/issues/10725#note-11 .

**Comment by smithfarm** [comment]
@tchaikov I think the make-check bot is paused, see Loic's email to ceph-devel delivered to me today 2015-06-03 at 01:48 CEST.

**Comment by tchaikov** [comment]
> @tchaikov I think the make-check bot is paused, see Loic's email to ceph-devel delivered to me today 2015-06-03 at 01:48 CEST.

okay. would you mind pushing it to ceph repo to get it built by gitbuilders?

File: doc/man/8/ceph-create-keys.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| **ceph-create-keys** [-h] [-v] [--cluster *name*] --id *id*
+
+
+Description
+===========
+
+:program:`ceph-create-keys` is a utility to generate bootstrap keyrings using
+the given monitor when it is ready.
+
+it creates following auth entities (or users)
```
Comment (theanalyst) [inline doc/man/8/ceph-create-keys.rst]: Possibly `It`

Comment (theanalyst) [inline doc/man/8/ceph-create-keys.rst]: Sorry didn't notice that this was a backport PR.. Guess it should be fine either way

Comment (tchaikov) [inline doc/man/8/ceph-create-keys.rst]: @theanalyst might be a chance to fix it though. we could add a note in the commit message for this small edit.

Comment (ktdreyer) [inline doc/man/8/ceph-create-keys.rst]: as long as it gets fixed in master too so it doesn't get lost :)

Comment (ktdreyer) [inline doc/man/8/ceph-create-keys.rst]: ... or should I say "It" doesn't get lost! :laughing:

Comment (tchaikov) [inline doc/man/8/ceph-create-keys.rst]: https://github.com/ceph/ceph/pull/4855

**Comment by smithfarm** [comment]
@tchaikov: rebased to include fixes from #4855

**Comment by tchaikov** [comment]
@smithfarm i am sure if i follow you, but the cherry-pick from 27cee2f is missing some bits:
https://github.com/SUSE/ceph/commit/27cee2f1f46a9f47cda9dfeb56ff1259e982960c#diff-ca40628478ff1274b73dcd40c384e7f0R640

**Comment by smithfarm** [comment]
@tchaikov: added the missing bit in `ceph.spec.in` - what else is missing?

**Comment by tchaikov** [comment]
@smithfarm  no, thanks! `Reviewed-by: Kefu Chai <kchai@redhat.com>` and also i pushed to ceph.git/wip-4846.

**Comment by ktdreyer** [comment]
LGTM

`Reviewed-by: Ken Dreyer <kdreyer@redhat.com>`

**Pull request merged**